Diagnostics/Get-NetView.PS1

```diff
@@ -1605,53 +1607,70 @@ function Completion {
         [parameter(Mandatory=$true)] [String] $Dest,
         [parameter(Mandatory=$true)] [String] $ZipName
     )
-    
+
     # Calculate sizes
     $dirs     = (Get-ChildItem $src -recurse | Measure-Object -property length -sum)
     $dirssize = "{0:N2}" -f ($dirs.sum / 1MB) + " MB"
 
     # Display file save location
-    
+
     Write-Host "`n`n"
     Write-Host "Diagnostics Data:"
     Write-Host "-----------------"
     Write-Host $MyInvocation.PSCommandPath
     Write-Host "Version: $version"
     Write-Host "SHA256:  $((Get-FileHash -Path $($MyInvocation.PSCommandPath) -Algorithm ""SHA256"").Hash)`n"
-    CreateZip -Src $Src -Dest $Dest -ZipName $ZipName 
+    CreateZip -Src $Src -Dest $Dest -ZipName $ZipName
     Write-Host "`n$Src"
     Write-Host "Size:   $dirssize"
     Write-Host "Dirs:  "(Get-ChildItem $Src -Directory -Recurse | Measure-Object | %{$_.Count})
     Write-Host "Files: "(Get-ChildItem $Src -File -Recurse | Measure-Object | %{$_.Count})
 } # Completion
 
 function Worker {
+    [CmdletBinding()]
+    Param(
+        [parameter(Mandatory=$false)] [String] $OutputDirectory
+    )
+
     clear
     $columns = 4096
```

Comment:
Move the -$version at the top of file to here.  Next to the locals.  No need to have it disjoint.